server/apps/main/views.py

```diff
@@ -502,3 +508,20 @@ def moderate_experience(request, uuid):
             )
     else:
         return redirect("index")
+
+
+def single_story(request, uuid):
```

Comment:
I guess we always expect there to be only a single experience so the `filter` and passing a list might be a bit overkill. 

A simpler solution could be a try/except: 

```
try:
  experience = PublicExperience.objects.get(experience_id=uuid,moderation_status='approved')
  … make context
  return render(…)
except DoesNotExist:
  return redirect(…)
```

That way we don't have to check for the length etc. and have a slightly clearer code?

Reply:
Thats nice and neat! Will implement